src/main/java/cn/nukkit/level/chunk/UnsafeChunk.java

```diff
@@ -236,11 +236,8 @@ public void setBlockData(int x, int y, int z, int layer, int data) {
         checkBounds(x, y, z);
         ChunkSection section = this.getSection(y >> 4);
         if (section == null) {
-            if (data == 0) {
-                // Setting air in an empty section.
-                return;
-            }
-            section = this.getOrCreateSection(y >> 4);
+            //setting meta on non-existing block?
```

Comment:
Maybe we should throw an exception here instead of ignoring it?

Reply:
Added an IllegalStateException